emcc.py

```diff
@@ -2546,7 +2565,8 @@ def get_eliminate():
 
       if (options.separate_asm or shared.Settings.WASM) and not shared.Settings.WASM_BACKEND:
         separate_asm_js(final, asm_target)
-        generated_text_files_with_native_eols += [asm_target]
+        if not shared.Settings.SINGLE_FILE:
```

Comment:
why was this condition added?

Reply:
In SINGLE_FILE we don't generate the .asm.js file, so that does not need to be added. `--output_eol` never worked with `-s SINGLE_FILE=1` before, just noticed that with MINIMAL_RUNTIME becauase we do --output_eol linux in those tests.